docs/configuration/settings.md

```diff
@@ -0,0 +1,190 @@
+
+### `FIELDS`
+
+All metadata fields are linked with their respective identifiers in the SOLR index in this hash. To learn more about the default GeoBlacklight metadata schema, Aardvark, see [Metadata](../../overview/metadata) or read the [OGM Aardvark specification](https://opengeometadata.org/ogm-aardvark/).
+
+### `INSTITUTION`
+
+This setting should hold the name of your institution, and can be used to help determine access to restricted records. In some GeoBlacklight implementations, for example, when a restricted record's `schema_provider_s` field matches `Settings.INSTITUTION`, authenticated users will be granted full access.
+
+### `METADATA_SHOWN`
+
+Enables links for various metadata formats in the tool panel for a record if the corresponding URI key is present in that record's `dct_references_s` field.
+
+### `TIMEOUT_DOWNLOAD`
+
+(For external Download) `timeout` and `open_timeout` parameters for Faraday.
+
+### `TIMEOUT_WMS`
+
+(For WMS inspection) `timeout` and `open_timeout` parameters for Faraday.
+
+### `USE_GEOM_FOR_RELATIONS_ICON`
+
+Use the Geometry Type value from a record to determine what icon to use for its data relations.
+
+!!! warning
+    This setting is only applicable for GBL 1.0 metadata and is not compatible with OGM Aardvark.
+
+### `WEBSERVICES_SHOWN`
+
+A list of web services that will be available for a record, if that record has a corresponding URI key in its `dct_references_s` field.
+
+For example, if a record's references include a `wms` entry, and `WEBSERVICES_SHOWN` includes `wms` (as it does by default), a preview map will appear in the tool panel showing the WMS layer.
+
+Supported web services:
+
+```
+  - 'wms'
+  - 'tms'
+  - 'wfs'
+  - 'xyz'
+  - 'wmts'
+  - 'tilejson'
+  - 'iiif'
+  - 'feature_layer'
+  - 'tiled_map_layer'
+  - 'dynamic_map_layer'
+  - 'image_map_layer'
+```
+
+### `DISPLAY_NOTES_SHOWN`
+
+Configuration for special rendering of `gbl_displayNote_sm` field values. Default note types are `danger`, `info`, `tip`, and `warning`.
+
+You can add your own display note configuration as well. Each entry must have the follow properties:
+
+bootstrap_alert_class
+: Name of Bootstrap alert class to use for the note's container
+
+icon
+: Name of GeoBlacklight SVG icon to display with note
+
+note_prefix
+: String that will be used at the beginning of a `gbl_displayNote_sm` entry to trigger this particular rendering.
+
+For example, the "info" note is configured like this:
+
+```
+DISPLAY_NOTES_SHOWN
+  info:
+    bootstrap_alert_class: alert-info
+    icon: circle-info-solid
+    note_prefix: "Info: "
+```
+
+!!! info
+    Display Notes will appear in GeoBlacklight in a similar manner to this admonition box.
+
+
+### `RELATIONSHIPS_SHOWN`
+
+GeoBlacklight supports many different types of relations between records. Configuration for how these are displayed is stored here. Each relationship defined must have the following properties:
+
```

Comment:
Oh! One small addition here... I've added an additional property: "inverse", which points to the inverse relation key/query. This will help us provide those browse links in v4.2. 

  SOURCE_DESCENDANTS:
    field: dct_source_sm
    icon: child-item
    inverse: :SOURCE_ANCESTORS
    label: geoblacklight.relations.source_descendants
    query_type: descendants

Reply:
Ok, and that value would be the name of a different relationship config?

Reply:
Yes! Exactly that.

Reply:
Ok, I've added that property now.